hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -339,6 +329,16 @@
       reset_connections: {rst_ni: "spi_host1"},
       base_addr: "0x40310000",
     },
+    { name: "usbdev",
+      type: "usbdev",
+      clock_srcs: {clk_i: "usb", clk_aon_i: "aon", clk_usb_48mhz_i: "usb"},
+      clock_group: "peri",
+      reset_connections: {rst_ni: "usb", rst_aon_ni: "sys_aon", rst_usb_48mhz_ni: "usbif"},
```

Comment:
should we take this opportunity to fix the `rst_aon_ni` connection? 
Maybe another PR would be better.